Mathlib/Topology/MetricSpace/Closeds.lean

```diff
@@ -397,7 +397,6 @@ theorem lipschitz_infDist_set (x : α) : LipschitzWith 1 fun s : NonemptyCompact
     exact infDist_le_infDist_add_hausdorffDist (edist_ne_top t s)
 
 theorem lipschitz_infDist : LipschitzWith 2 fun p : α × NonemptyCompacts α => infDist p.1 p.2 := by
-  -- Porting note: Changed tactic from `exact` to `convert`, because Lean had trouble with 2 = 1 + 1
```

Comment:
Should we be satisfied with exact not working here?
Or should this morally be a `simpa using` proof?
`convert` is a bit of a crowbar.

Reply:
`simpa using` it timing out here.